CMakeLists.txt

```diff
@@ -11,9 +11,9 @@ if (NOT NRF5_SDK_PATH)
   message(FATAL_ERROR "The path to the NRF52 SDK must be specified on the command line (add -DNRF5_SDK_PATH=<path>")
 endif ()
 
-if(NOT USE_JLINK AND NOT USE_GDB_CLIENT)
-  set(USE_JLINK true)
-endif()
+#if(NOT USE_JLINK AND NOT USE_GDB_CLIENT)
+#  set(USE_JLINK true)
+#endif()
```

Comment:
The goal of this condition was to ensure that a default behavior (jlink) was enabled if no USE_* variable was specified.

We could either keep it or remove it:
 - if we want to keep it, we should add USE_OPENOCD in the condition
 - if we do not keep it, we must ensure that the project builds correctly and update the documentation (no USE_* means that the project builds but does not provide support for flashing, or one of the USE_* variable is defined, and in this case, the project builds and provide targets to flash the firmware in the device.

What do you think?

Reply:
I think the first alternative works, I just forgot to review the code twice

Reply:
For now, I think it's enough. We'll refactor this part if we need to add support for more debugger. Thanks for your pull request!